src/translations/screens/Profile.ts

```diff
@@ -3,8 +3,8 @@ import orgSpecificTranslations from '../utils';
 
 const ProfileTexts = {
   header: {
-    title: _('Mitt AtB', 'My AtB'),
-    title_a11y: _('Mitt ATB', 'My ATB'),
+    title: _('Min profil', 'My profile'),
+    title_a11y: _('Min profil', 'My profile'),
```

Comment:
Behøver ikke disse `title_a11y` lenger. De var tidligere nødvendig siden "AtB" ikke ble lest opp bokstav for bokstav, derfor måtte man ha "ATB" i egen a11y-tekst.

Reply:
Den er god! Oppdaterer.